packages/network/tests/thor-client/contracts/contract.erc20.solo.test.ts

```
@@ -1,5 +1,12 @@
 import { beforeEach, describe, expect, test } from '@jest/globals';
-import { Address, ERC20_ABI, HexUInt } from '@vechain/sdk-core';
+import {
+    ABIFunction,
```

ESLint

[error] 'ABIFunction' is defined but never used.

(@typescript-eslint/no-unused-vars)